app/src/main/kotlin/cz/covid19cz/erouska/crypto/keystore.kt

```diff
@@ -0,0 +1,93 @@
+package cz.covid19cz.erouska.crypto
+
+import android.content.Context
+import android.security.KeyPairGeneratorSpec
+import cz.covid19cz.erouska.db.SharedPrefsRepository
+import java.math.BigInteger
+import java.nio.charset.Charset
+import java.security.*
+import java.util.*
+import javax.crypto.Cipher
+import javax.security.auth.x500.X500Principal
+
+private const val ENCRYPTION_ALGORITHM = "RSA/ECB/PKCS1Padding"
+
+// Must be <= 256 bytes
+private const val DATABASE_KEY_SIZE = 32
+
+const val DATABASE_KEY_ALIAS = "db-key"
+
+fun getDatabaseEncryptionKey(prefsRepository: SharedPrefsRepository, context: Context): ByteArray? {
```

Comment:
Global methods accessible from anywhere are not good. Either make them extension methods or wrap them in `object`

Reply:
I see nothing wrong with them, but I will do as you ask! :-)